lms/djangoapps/verify_student/models.py

```diff
@@ -721,8 +721,8 @@ def submit(self, copy_id_photo_from=None):
                 self.status = "must_retry"
```

Comment:
Can you first then what can be the possibilities for a response is not Ok? I mean what will be the scenario of getting `response.ok == False`